app/konnect/dev-portal/access-and-approval/manage-teams.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Manage Dev Portal Teams
+content_type: how-to
+---
+
+A common scenario for a Dev Portal is to restrict developers' ability to view or request access to specific API products based on their permissions. {{site.konnect_short_name}} Portal enables administrators to define Role-Based Access Control (RBAC) for teams of developers through the Konnect UI and API.
+
+Portal RBAC supports two different roles for API products that can be applied to a team of developers:
+
+* **API Viewer:** Provides developers with read access to the documentation associated with an API product.
+* **API Consumer:** Includes API Viewer permissions as well as allows developers to register applications to consume versions of an API product.
+
+You can use the API and UI to create a team, assign a role to the team, and finally, add developers to the team.
+
+In this guide, you will set up two developer teams and then enable Portal RBAC using a hypothetical scenario.
+
+## Prerequisites
+* Two [test developer accounts registered](/konnect/dev-portal/dev-reg/). If you currently have developers in {{site.konnect_short_name}}, you must configure developer teams before enabling Portal RBAC (which is disabled by default). If you enable Portal RBAC before configuring teams for your developers, it will prevent *all* developers from seeing any API products in your Dev Portal. We recommend setting up your teams and permissions before enabling RBAC which will allow for a seamless transition; developers see what they're supposed to, instead of nothing at all.
+* A [runtime configured](/konnect/getting-started/configure-runtime)
+* A [personal access token](/konnect/getting-started/import/#generate-a-personal-access-token) for authorization
```

Comment:
We should note that PAT is only required if you plan on doing this tutorial with the API.  UI will not require PAT.